app/scripts/HiGlassComponent.js

```diff
@@ -563,12 +563,51 @@ class HiGlassComponent extends React.Component {
     this.pixiRenderer.render(this.pixiStage);
   }
 
+  /**
+   * Add default track options. These can come from two places:
+   *
+   * 1. The track definitions (options/tracks-info.js)
+   * 2. The default options passed into the component
+   *
+   * Of these, #2 takes precendence over #1.
+   *
+   * @param {array} track The track to add default options to
+   */
   addDefaultOptions(track) {
     const trackInfo = this.getTrackInfo(track.type);
     if (!trackInfo) return;
 
+    if (typeof track.options === 'undefined') {
+      track.options = {};
+    }
+
     const trackOptions = track.options ? track.options : {};
 
+    if (this.props.options.defaultOptions) {
+      if (this.props.options.defaultOptions.trackSpecific
+        && this.props.options.defaultOptions.trackSpecific[track.type]) {
+        // track specific options take precedence over all options
+
+        const options = this.props.options.defaultOptions.trackSpecific[track.type];
+
+        for (const optionName in options) {
+          track.options[optionName] = typeof (track.options[optionName]) !== 'undefined'
```

Comment:
Brackets after `typeof` are not necessary. `track.options[optionName]` is already a single expression